chapters/publication.tex

```diff
@@ -99,91 +174,259 @@ \section{Collaborating on academic writing}
-
-Data and code should always be released with any publication.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Publishing_Data}}
-\index{data publication}
-Many journals and funders have strict open data policies,
-and providing these materials to other researchers
-allows them to evaluate the credibility of your work
-as well as to re-use your materials for further research.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Exporting_Analysis}}
-If you have followed the steps in this book carefully,
-you will find that this is a very easy requirement to fulfill.\sidenote{
-\url{https://www.bitss.org/2016/05/23/out-of-the-file-drawer-tips-on-prepping-data-for-publication/}}
-You will already have a publishable (either de-identified or constructed)
-version of your research dataset.
-You will already have your analysis code well-ordered,
-and you won't have any junk lying around for cleanup.
-You will have written your code so that others can read it,
-and you will have documentation for all your work,
-as well as a well-structured directory containing the code and data.
+\section{Preparing a complete replication package}
+
+While we have focused so far on the preparation of written materials for publication,
+it is increasingly important for you to consider how you will publish
+the data and code you used for your research as well.
+More and more major journals are requiring that publications
+provide direct links to both the code and data used to create the results,
+and some even require being able to reproduce the results themselves
+before they will approve a paper for publication.\sidenote{
+  \url{https://www.aeaweb.org/journals/policies/data-code/}}
+If your material has been well-structured throughout the analytical process,
+this will only require a small amount of extra work;
+if not, paring it down to the ``replication package'' may take some time.
+A complete replication package should accomplish several core functions.
+It must provide the exact data and code that is used for a paper,
+all necessary de-identified data for the analysis,
+and all code necessary for the analysis.
+The code should exactly reproduce the raw outputs you have used for the paper,
+and should include no extraneous documentation or PII data you would not share publicly.
+
+\subsection{Publishing data for replication}
+
+Enabling permanent access to the data used in your study
+is an important contribution you can make along with the publication of results.
+It allows other researchers to validate the mechanical construction of your results,
+to investigate what other results might be obtained from the same population,
+and test alternative approaches to other questions.
+Therefore you should make clear in your study
+where and how data are stored and how it might be accessed.
+You do not have to publish data yourself,
+although in many cases you will have the right to release
+at least some subset of your analytical dataset.
```

Comment:
I do not understand what we are trying to say here:

_You do not have to publish data yourself, although in many cases you will have the right to release at least some subset of your analytical dataset._

Reply:
Much more detail b73d2d0a46eda52016c1e3b9973cc2b6d484f69b

Reply:
ok, now I understand what we are trying to say, What about these suggestions broken down to each point we are trying to make:

"_You do not always have to publish the data yourself, and in some cases you are legally not allowed to, but what matters is that the data is published (with or without access restrictions) and that you cite or otherwise directly reference all data, even data that you cannot release._"

Constructed is bold as I changed analytical to that. What we used to describe this data set in the previous chapter was constructed.

"_Even when your raw data is owned by someone else, or for any other reason are not able to publish it, in many cases you will have the right to release at least some subset of your **constructed** data set even if it is just indicators you constructed._"

Can this be re-written. I understand what you are getting at, but not exactly which points you are trying to make.
"_Check with the data supplier or other professional about licensing questions,
particularly your right to publish derivative materials._"

Reply:
80ad7f6559caeea43f26c66cdba5499cd3712dc4